src/component/folder/FolderContain.js

```diff
@@ -0,0 +1,31 @@
+﻿import { useEffect, useState } from "react";
+import { getUserFolders } from "../../api/getUserFolders";
+import FolderLists from "./FolderLists";
+
+function FolderList() {
+  const [folders, setFolders] = useState([]);
+
+  useEffect(() => {
+    async function getFolderData() {
+      const x = await getUserFolders();
+      setFolders(x.data);
```

Comment:
```suggestion
      const { data: folders } = await getUserFolders();
      setFolders(folders);
```

x처럼 예측할 수 없는 변수명은 지양해주세요.